pinecone/sparse/bm25/bm25-vector-generation.ipynb

```diff
@@ -0,0 +1,6251 @@
+{
```

Comment:
<p>Capitalization: [F]or more details...</p><p>Comma after "on that[,]..."</p><p>Two words: "checkout" -&gt; "check out"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>